core/src/services/routing.js

```diff
@@ -236,6 +237,7 @@ export const handleRouteChange = async (path, component, node, config) => {
       pathParams,
       isolateView,
       viewGroup,
+      pathUrlRaw,
```

Comment:
As the only thing we are doing with `patUrlRaw` is extract the url params in _App.html_, why don't we just save the url params instead?